external/loaders/loaders/mappers/_fine_resolution_budget.py

```diff
@@ -434,7 +432,7 @@ def open_fine_res_apparent_sources(
         "delp": "pressure_thickness_of_atmospheric_layer",
     }
     dim_order = ("tile", "z", "y", "x")
-    drop_vars = ("step", "time")
+    drop_vars = ("step",)
```

Comment:
Did you want to also drop time in this mapper given that @brianhenn said it was alright?

Reply:
Yes I want to stop dropping time in this mapper.